uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeExpiryLogTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChangeExpiryLog as LeaveBalanceChangeExpiryLog;
+use CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChange as LeaveBalanceChange;
+/**
+ * Class CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChangeExpiryLogTest
+ *
+ * @group headless
+ */
+class CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChangeExpiryLogTest extends BaseHeadlessTest {
+
+  /**
+   * @expectedException CRM_HRLeaveAndAbsences_Exception_InvalidLeaveBalanceChangeExpiryLogException
+   * @expectedExceptionMessage The balance_change_id field should not be empty
+   */
+  public function testCreateLeaveBalanceChangeExpiryLogThrowsExceptionWhenBalanceChangeIdIsAbsent() {
```

Comment:
Since the text is in the context of the BAO, I think we don't need to repeat the BAO name in the method. So, instead of `testCreateLeaveBalanceChangeExpiryLog` you can just say `testCreate`